README.md

```diff
@@ -0,0 +1,107 @@
+# GenAI Factory
+
+Demo an end to end LLM agent solution with modular architecture, persistent storage and front-end UI that can work with various LLM models and storage solutions.
+
+the configuration is specified in a YAML file, which indicate the model, embeddings, storage to use, and various parameters. 
+the user can point to the configuration file by setting the `AGENT_CONFIG_PATH` environment variable.
+
+environment variables and credentials can be loaded from a `.env` file in the root directory. or an alternate path set by the `AGENT_ENV_PATH` environment variable.
+data can be stored in local files or remote SQL and Vector databases. the local file storage path can be set by the `AGENT_DATA_PATH` environment variable (defaults to `./data/`).
+
+# Example configuration
+
+{see the AppConfig class in the src/config.py file}
+```yaml
+chunk_overlap: 20
+chunk_size: 1024
+default_llm:
+  class_name: langchain.chat_models.ChatOpenAI
+  model_name: gpt-3.5-turbo
+  temperature: 0
+default_vector_store:
+  class_name: chroma
+  collection_name: default
+  persist_directory: C:\Users\Yaron Haviv\PycharmProjects\demo-llm-agent\data\chroma
+embeddings:
+  class_name: huggingface
+  model_name: all-MiniLM-L6-v2
+log_level: DEBUG
+pipeline_args: {}
+verbose: false
+```
```

Comment:
this is an application config (even points to AppConfig in line 13). Should be moved to a README in the branch `genai` of MLRun.